packages/synapse-react-client/src/lib/style/abstracts/_variables.scss

```diff
@@ -120,12 +120,17 @@ $breakpoints: (
   'large': 1024px,
 ) !default;
 
+// Buttons
+$button-border-radius: 3px;
+$button-text-transform: capitalize;
+
 // To reset bootstrap button style
 %button-reset {
   border: 0;
   outline: none;
   box-shadow: none;
-  border-radius: 2px;
+  border-radius: $button-border-radius;
+  text-transform: $button-text-transform;
```

Comment:
Specified for PageProgress, which extends `%button-reset` in `_page-progress.scss`
main | feature
:----:|:----:
<img width="703" alt="main_PageProgress" src="https://user-images.githubusercontent.com/26949006/231309044-b3300564-127a-49bc-b1ed-009137f2f67c.png"> | <img width="664" alt="feature_PageProgress" src="https://user-images.githubusercontent.com/26949006/231309025-baf779c3-5360-4054-aa9c-07cb17fb5d92.png">